openml/datasets/functions.py

```diff
@@ -333,6 +333,27 @@ def _load_features_from_file(features_file: str) -> Dict:
         return xml_dict["oml:data_features"]
 
 
+def _expand_parameter(parameter: Union[str, list]):
+    expanded_parameter = []
+    if isinstance(parameter, str):
+        expanded_parameter = [x.strip() for x in parameter.split(",")]
+    elif isinstance(parameter, list):
+        expanded_parameter = parameter
+    return expanded_parameter
+
+
+def _validated_data_attributes(attributes: list, data_attributes: list, parameter_name: str):
```

Comment:
So here it should also say `-> None`. I am not adding it as a commit suggestion here, because I think it would violate the line length constraint we have. So best to do this manually and then have the `pre-commit` hook re-format if necessary.